packages/api/src/platforms/vtex/clients/search/index.ts

```diff
@@ -66,15 +66,17 @@ export const IntelligentSearch = (
   }
 
   const getRegionFacet = (): IStoreSelectedFacet | null => {
-    const { regionId } = ctx.storage.channel
+    const { regionId, seller } = ctx.storage.channel
+    const sellerRegionId = seller ? Buffer.from(`SW#${seller}`).toString('base64') : null
```

Comment:
out of curiosity. from where comes this Buffer logic?

Reply:
Oh, it's a native JS feature :)

Reply:
I mean, regarding the `SW#`?